test/unit/HTTP_reveice_measures-test.js

```diff
@@ -1,339 +0,0 @@
-                contextBrokerMock.done();
-                done();
-            });
-        });
-    });
-
-    describe('When a POST measure arrives for an unprovisioned device', function() {
-        const optionsMeasure = {
-            url: 'http://localhost:' + config.http.port + '/iot/json',
-            method: 'POST',
-            json: {
-                humidity: '32',
-                temperature: '87'
-            },
-            headers: {
-                'fiware-service': 'smartGondor',
-                'fiware-servicepath': '/gardens'
-            },
-            qs: {
-                i: 'MQTT_UNPROVISIONED',
-                k: 'KL223HHV8732SFL1'
-            }
-        };
-
-        beforeEach(function(done) {
-            contextBrokerUnprovMock = nock('http://unexistentHost:1026')
-                .matchHeader('fiware-service', 'smartGondor')
-                .matchHeader('fiware-servicepath', '/gardens')
-                .post('/v1/updateContext')
-                .reply(200, utils.readExampleFile('./test/contextResponses/multipleMeasuresSuccess.json'));
-
-            contextBrokerUnprovMock
-                .matchHeader('fiware-service', 'smartGondor')
-                .matchHeader('fiware-servicepath', '/gardens')
-                .post('/v1/updateContext', utils.readExampleFile('./test/contextRequests/unprovisionedDevice.json'))
-                .reply(200, utils.readExampleFile('./test/contextResponses/multipleMeasuresSuccess.json'));
-
-            request(groupCreation, function(error, response, body) {
-                done();
-            });
-        });
-
-        it('should send its value to the Context Broker', function(done) {
-            request(optionsMeasure, function(error, result, body) {
-                contextBrokerUnprovMock.done();
-                done();
-            });
-        });
-        it('should add a transport to the registered devices', function(done) {
-            const getDeviceOptions = {
-                url: 'http://localhost:' + config.iota.server.port + '/iot/devices',
-                method: 'GET',
-                headers: {
-                    'fiware-service': 'smartGondor',
-                    'fiware-servicepath': '/gardens'
-                },
-                qs: {
-                    i: 'MQTT_UNPROVISIONED',
-                    k: 'KL223HHV8732SFL1'
-                }
-            };
-
-            request(optionsMeasure, function(error, response, body) {
-                request(getDeviceOptions, function(error, response, body) {
-                    should.not.exist(error);
-                    const parsedBody = JSON.parse(body);
-
-                    response.statusCode.should.equal(200);
-                    should.exist(parsedBody.devices[0].transport);
-                    parsedBody.devices[0].transport.should.equal('HTTP');
-                    done();
-                });
-            });
-        });
```

Comment:
This "it" scenario is not checked on [V2 test file equivalent](https://github.com/telefonicaid/iotagent-json/blob/master/test/unit/ngsiv2/HTTP_reveice_measures-test.js), could you adapt it?

Reply:
Fixed 45e83ef